torch/fx/graph.py

```diff
@@ -121,14 +121,16 @@ def forward(self, x):
 
     For the semantics of operations represented in the ``Graph``, please see :class:`Node`.
     """
-    def __init__(self):
+    def __init__(self, owning_module: Optional["GraphModule"] = None):     # type: ignore[name-defined]
```

Comment:
What's the typing comment here for?

Reply:
mypy error: `torch/fx/graph.py:124: error: Name 'GraphModule' is not defined  [name-defined]`. Note that this is with the forward reference `"GraphModule"`

Reply:
Does it work if we gate an import of `GraphModule` with `if TYPE_CHECKING`? e.g. https://github.com/pytorch/pytorch/blob/64847c7f0b3559c6edc40f001619b80c7dc68ef7/torch/fx/node.py#L8